ckanext/bcgov/util/util.py

```diff
@@ -475,3 +475,9 @@ def can_view_resource(resource):
                 return True
 
     return False
+
+def get_package_tracking(package):
+    return (model.TrackingSummary.get_for_package(package['id']))
```

Comment:
Pass in just the `id` var rather then the dataset dict.

Reply:
done.